pkg/volume/glusterfs/glusterfs.go

```diff
@@ -211,6 +212,13 @@ func (b *glusterfsMounter) GetAttributes() volume.Attributes {
 // to mount the volume are available on the underlying node.
 // If not, it returns an error
 func (b *glusterfsMounter) CanMount() error {
+	exe := exec.New()
+	switch runtime.GOOS {
+	case "linux":
+		if _, err := exe.Command("/bin/ls", "/sbin/mount.glusterfs").CombinedOutput(); err != nil {
```

Comment:
Please define the `mount client binary path` as constant. It would also be good if you can add a comment about `what distros` are verified  under `linux` .